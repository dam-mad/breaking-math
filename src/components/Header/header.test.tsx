//TODO: I need to set up testing for typescript

// import React from 'react';
// import { render } from '@testing-library/react';
// import Header from './header';

// describe('Header', () => {
//   it('renders correctly', () => {
//     const { container } = render(<Header siteTitle="Default Starter" />);
//     expect(container).toMatchSnapshot();
//   });
// });
