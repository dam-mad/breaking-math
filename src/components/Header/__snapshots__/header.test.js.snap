// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa iDGsLm"
  >
    <header>
      <div>
        <h1>
          <a
            href="/"
            style="color: white; text-decoration: none;"
          >
            Default Starter
          </a>
        </h1>
      </div>
    </header>
  </div>
</div>
`;
